content/docs/csidriver/features/powermax.md

```diff
@@ -678,3 +678,27 @@ These are the CSM modules not supported with NVMeTCP protocol:
 - CSM Observability
 - CSM Application Mobility
 - Metro Replication
+
+## Mount Credentials Support
+
+From version 2.14.0, the CSI Driver for Dell PowerMax supports Mount Credentials, making the configMap obsolete. In earlier versions of the PowerMax Reverse Proxy, configuring it required creating a config map that referenced credentials stored in secrets. Now, a single mountable secret can be used to store both array information and credentials, simplifying the setup.
+```yaml
+storageArrays:
+  - storageArrayId: "000000000001"
+    primaryEndpoint: https://primary-1.unisphe.re:8443
+    backupEndpoint: https://backup-1.unisphe.re:8443
+managementServers:
+  - endpoint: https://primary-1.unisphe.re:8443
+    username: admin
+    password: password
+    skipCertificateValidation: true
+  - endpoint: https://backup-1.unisphe.re:8443
+    username: admin2
+    password: password2
+    skipCertificateValidation: false
+    certSecret: primary-cert
```

Comment:
Also, can we add limits to depict its support as well (could be added to primary, limits at the same level as the other keys) -
limits:
      maxActiveRead: 5
      maxActiveWrite: 4
      maxOutStandingRead: 50
      maxOutStandingWrite: 50